README.md

```diff
@@ -247,7 +247,7 @@ After a few minutes, all resources should be deployed.
 **Credentials for the 'admin' user can be found in the `./deploy/k8s/central-deploy/password` file.**
 
 > **Note:**
-> While the password file is stored in plaintext on your local filesystem, the Kubernetes Secret StackRox uses is encrypted, and you will not be able to alter the secret at runtime. If you lose the password, you will have to redeploy central.
+> The password file is stored in plaintext on your local filesystem, but the Kubernetes Secret StackRox creates from it is encrypted, and you will not be able to alter it at runtime. If you lose the password, you will have to redeploy central.
```

Comment:
Much better! Here is small addition 


```suggestion
> The password file is stored in plaintext on your local filesystem, but the Kubernetes Secret that StackRox creates from it is encrypted. You will not be able to alter the password at runtime. If you lose the password, you will have to redeploy central.
```

Additionally here is how I would write it using more passive voice to do less finger pointing.

```suggestion
> The password file is stored in plaintext on your local filesystem, but the Kubernetes Secret that StackRox creates from it is encrypted. It is impossible to change the password at runtime after it is created. If the password is lost, central needs to be redeployed.
```

Reply:
Thanks! Your first suggestion matches the style in the rest of the README so I used that.